projects/1.20/assets/eternal-tales/eternal_tales/lang/zh_cn.json

```
@@ -1026,36 +1026,36 @@
 	"block.eternal_tales.destroyers_eye_block": "毁灭者之眼块",
 	"block.eternal_tales.diamond_lamp": "钻石灯",
 	"block.eternal_tales.diamondbricks": "钻石砖",
-	"block.eternal_tales.dice_block": "Dice Block",
+	"block.eternal_tales.dice_block": "骰子块",
 	"block.eternal_tales.digging_skill_stone": "挖掘技能石",
 	"block.eternal_tales.digging_skill_stone_active": "已激活的挖掘能力石头",
 	"block.eternal_tales.diorite_dirt": "闪长岩泥土",
 	"block.eternal_tales.djinn_lamp_block": "神灯",
 	"block.eternal_tales.double_chain": "双层锁链",
 	"block.eternal_tales.double_gold_chain": "双层金锁链",
 	"block.eternal_tales.double_unahzaal_chain": "双层乌纳扎尔锁链",
-	"block.eternal_tales.dredert_bottom": "Dredert",
-	"block.eternal_tales.dredert_button": "Dredert Button",
-	"block.eternal_tales.dredert_door": "Dredert Door",
-	"block.eternal_tales.dredert_fence": "Dredert Fence",
-	"block.eternal_tales.dredert_fence_gate": "Dredert Fence Gate",
-	"block.eternal_tales.dredert_leaves": "Dredert Leaves",
-	"block.eternal_tales.dredert_log": "Dredert Log",
-	"block.eternal_tales.dredert_planks": "Dredert Planks",
-	"block.eternal_tales.dredert_pressure_plate": "Dredert Pressure Plate",
-	"block.eternal_tales.dredert_slab": "Dredert Slab",
-	"block.eternal_tales.dredert_stairs": "Dredert Stairs",
-	"block.eternal_tales.dredert_top": "Dredert",
-	"block.eternal_tales.dredert_trapdoor": "Dredert Trapdoor",
-	"block.eternal_tales.dredert_wood": "Dredert Wood",
+	"block.eternal_tales.dredert_bottom": "恐怖木",
+	"block.eternal_tales.dredert_button": "恐怖木按钮",
+	"block.eternal_tales.dredert_door": "恐怖木门",
+	"block.eternal_tales.dredert_fence": "恐怖木栅栏",
+	"block.eternal_tales.dredert_fence_gate": "恐怖木栅栏门",
+	"block.eternal_tales.dredert_leaves": "恐怖树叶",
+	"block.eternal_tales.dredert_log": "恐怖木原木",
```

```suggestion
	"block.eternal_tales.dredert_log": "恐怖原木",
```
见#4525 *方便组合文件的用法（新增1）*

---

错了，这里是取决于dredert译作“恐怖（树）”还是“恐怖木（树）”
若译作恐怖，则按一般类变化 恐怖（树叶） 恐怖（原木） 恐怖木（楼梯）
若译作恐怖木，则按含木类变化  恐怖木 恐怖木 恐怖木
只是对树木译名变体的标准化，没有要求树木译名。